phpBB/config/installer/container/services_installer.yml

```diff
@@ -0,0 +1,70 @@
+imports:
+    - { resource: services_install_controller.yml }
+    - { resource: services_install_data.yml }
+    - { resource: services_install_database.yml }
+    - { resource: services_install_filesystem.yml }
+    - { resource: services_install_finish.yml }
+    - { resource: services_install_navigation.yml }
+    - { resource: services_install_obtain_data.yml }
+    - { resource: services_install_requirements.yml }
+
+services:
+# -------- Installer helpers ------------------------
+    installer.helper.config:
+        class: phpbb\install\helper\config
+        arguments:
+            - @filesystem
+            - @php_ini
+            - %core.root_path%
+
+    installer.helper.database:
+        class: phpbb\install\helper\database
+        arguments:
+            - @filesystem
+            - %core.root_path%
+
+    installer.helper.iohandler_factory:
+        class: phpbb\install\helper\iohandler\factory
+        arguments:
+            - @service_container
+
+    installer.helper.iohandler_abstract:
+        abstract: true
+        calls:
+            - [set_language, ["@language"]]
+
+    installer.helper.iohandler_ajax:
+        class: phpbb\install\helper\iohandler\ajax_iohandler
+        parent: installer.helper.iohandler_abstract
+        arguments:
+            - @request
+            - @template
+
+    installer.helper.iohandler:
+        class: phpbb\install\helper\iohandler\iohandler_interface
+        factory: ["@installer.helper.iohandler_factory", get]
+
+    installer.helper.container_factory:
+        class: phpbb\install\helper\container_factory
+        arguments:
+            - @request
+            - %core.root_path%
+            - %core.php_ext%
+
+# -------- Installer --------------------------------
+    installer.module_base:
+        abstract: true
+        calls:
+            - [setup, [@service_container, @installer.helper.config, @installer.helper.iohandler]]
```

Comment:
the services are lazy loaded and if you retrieve only the keys they shouldn't be loaded (currently it's not possible to retrieve an array of the keys but it can be done)